Lib/dis.py

```diff
@@ -581,7 +581,12 @@ def _disassemble_bytes(code, lasti=-1, varname_from_oparg=None,
                            instr.offset > 0)
         if new_source_line:
             print(file=file)
-        is_current_instr = instr.offset == lasti
+        if show_caches:
+            is_current_instr = instr.offset == lasti
+        else:
+            # Each CACHE takes 2 bytes
+            is_current_instr = instr.offset <= lasti \
+                <= instr.offset + 2 * _inline_cache_entries[_deoptop(instr.opcode)]
         print(instr._disassemble(lineno_width, is_current_instr, offset_width),
```

Comment:
What happens with lasti points to an EXTENDED_ARG?

Reply:
I don't see any changes there compared to the old behavior. Not sure if `EXTENDED_ARG` has any `CACHE` entries, but as long as the `CACHE` entries are 2 byte long, it should have the consistant behavior as before.

Reply:
EXTENDED_ARG doesn't have caches. I was thinking that it gets rolled into the next instruction, but it doesn't (dis shows it) so there shouldn't be any issues.